client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
It would be nice to place your custom PR.
It should include a description and instructions on how to run your app.